app/src/main/java/projekt/substratum/fragments/ManagerFragment.java

```diff
@@ -349,10 +349,10 @@ public void onClick(View view) {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.overlays_list_menu, menu);
-        menu.findItem(R.id.restart_systemui).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
-        menu.findItem(R.id.per_app).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
```

Comment:
I don't know if we want this

Reply:
You mean using "ifRoom" instead of "always"? It shouldn't make a difference on the average device. It's more to silence lint warnings

Reply:
Hmm interesting, will let @nicholaschum make the call then

Reply:
We want this set to always.